src/types/reader.types.ts

```diff
@@ -36,3 +36,14 @@ export enum StorageKey {
   carrotSettings = "carrotSettings",
   carrotTheme = "carrotTheme"
 }
+
+export interface Metadata {
```

Comment:
you should support fallback values / null values - i can definitely see cases where having an `original_title` field is n/a, or the author is unknown or sm.

so just make whichever of these fields that make sense nullable, and just check in the ui to display them or not

Reply:
Yeah so it turns out I don't need to make it nullable for it to be nullable - like the author title description I have not defined them for what we currently have and it's fine - I will add checks for undefined as mentioned above

Reply:
i keep mixing up nullable and undefined - i meant undefined. so it should look sm like this:
```
{
  title: string,
  original_title?: string,
  author?: string,
  description?: string,
  cover_image?: string,
  url: string,
  type: string,
  original: boolean,
} 
```